core/variable_operations.cpp

```diff
@@ -138,7 +138,60 @@ Variable filter(const Variable &var, const Variable &filter) {
   return out;
 }
 
+struct disable_variance_first_arg {
+  template <class First, class Second>
+  void operator()(const scipp::core::detail::ValueAndVariance<First>,
+                  const Second &) const {
+    throw except::VariancesError("Variances in first argument not supported.");
+  }
+};
+
+struct disable_variance_second_arg {
+  template <class First, class Second>
+  void operator()(const First &,
+                  const scipp::core::detail::ValueAndVariance<Second>) const {
+    throw except::VariancesError("Variances in first argument not supported.");
+  }
+};
+
+namespace sparse {
+/// Return array of sparse dimension extents, i.e., total counts.
+Variable counts(const VariableConstProxy &var) {
```

Comment:
I don't really see a better name than `counts` for this. It also makes sense since the `Variable` it will return has units of `counts`.
I thought of `sparse_content_size`, `sparse_size`, `counts_per_pixel`, `counts_per_element`, but none of it is satisfactory nor better than just `counts`...